ts/features/wallet/bancomat/component/bancomatCard/BancomatCard.tsx

```diff
@@ -25,22 +27,61 @@ const getExpireDate = (fullYear?: string, month?: string): Date | undefined => {
   return new Date(year, indexedMonth - 1);
 };
 
+/**
+ * Generate the accessibility label for the card.
+ */
+const getAccessibilityRepresentation = (
+  bankName: string,
+  expiringDate?: Date,
+  holder?: string
+) => {
+  const cardRepresentation = I18n.t("wallet.accessibility.folded.bancomat", {
+    bankName
+  });
+
+  const computedValidity =
+    typeof expiringDate !== "undefined"
+      ? `, ${I18n.t("cardComponent.validUntil")} ${localeDateFormat(
+          expiringDate,
+          I18n.t("global.dateFormats.numericMonthYear")
+        )}}`
+      : "";
+
+  const computedHolder =
+    typeof holder !== "undefined"
+      ? `, ${I18n.t("wallet.accessibility.cardHolder")} ${holder}`
+      : "";
+
+  return `${cardRepresentation}${computedValidity}${computedHolder}`;
+};
+
 /**
  * Render a bancomat already added to the wallet
  * @param props
  * @constructor
  */
-const BancomatCard: React.FunctionComponent<Props> = props => (
-  <BaseBancomatCard
-    abi={props.enhancedBancomat.abiInfo ?? {}}
-    isExpired={isPaymentMethodExpired(props.enhancedBancomat).getOrElse(false)}
-    expiringDate={getExpireDate(
-      props.enhancedBancomat.info.expireYear,
-      props.enhancedBancomat.info.expireMonth
-    )}
-    user={props.nameSurname ?? ""}
-  />
-);
+const BancomatCard: React.FunctionComponent<Props> = props => {
+  const expiringDate = getExpireDate(
+    props.enhancedBancomat.info.expireYear,
+    props.enhancedBancomat.info.expireMonth
+  );
+
+  return (
+    <BaseBancomatCard
+      accessibilityLabel={getAccessibilityRepresentation(
+        props.enhancedBancomat.caption,
+        expiringDate,
+        props.nameSurname
```

Comment:
Since we have the `BaseBancomatCard` used also from `PreviewBancomatCard` (wallet home -> add payment method -> bancomat -> the found user bancomat) what you think about moving the accessibility logic inside `BaseBancomatCard`(like you did for `BasePrivativeCard`), in order to make accessible all the usages of `BaseBancomatCard`?